File: InMotionGIT.Common - Copy/Services/Contracts/QueryResult.vb
Imports System.Runtime.Serialization

Namespace Services.Contracts

    <DataContract()>
    Public Class QueryResult

        <DataMember()>
        Public Property QueryCountResult As Integer

        <DataMember()>
        Public Property Table As DataTable

        <DataMember()>
        Public Property OutputParameters As New Dictionary(Of String, Object)

    End Class

End Namespace

File: Common/InMotionGIT.Common/Services/Contracts/QueryResult.cs
using System.Collections.Generic;
using System.Data;
using System.Runtime.Serialization;

namespace InMotionGIT.Common.Services.Contracts
{

    [DataContract()]
    public class QueryResult
    {

        [DataMember()]
        public int QueryCountResult { get; set; }

        [DataMember()]
        public DataTable Table { get; set; }

        [DataMember()]
        public Dictionary<string, object> OutputParameters { get; set; } = new Dictionary<string, object>();

    }

}